### PR TITLE
fix(pipelines): update running and pending icon colors

### PR DESCRIPTION
**Pull request by karthikjeeyar**
Fixes:
Bug  - https://github.com/patternfly/react-topology/issues/12

Screenshots:

Screenshot 2023-04-11 at 7 15 10 PM Screenshot 2023-04-11 at 6 56 36 PM

Screenshot 2023-04-11 at 6 57 00 PM Screenshot 2023-04-11 at 6 57 10 PM

### CODE REVIEW

File: packages/module/src/pipelines/components/nodes/TaskNode.tsx [hunk 1]
```diff
@@ -121,7 +121,16 @@ const TaskNode: React.FC<TaskNodeProps & { innerRef: React.Ref<SVGGElement> }> =
...
+  hiddenDetailsShownStatuses = [
+    RunStatus.Failed,
+    RunStatus.FailedToStart,
+    RunStatus.Cancelled,
+    RunStatus.Running,
+    RunStatus.Skipped,
+    RunStatus.Succeeded,
+    RunStatus.Pending,
+    RunStatus.InProgress
+  ],
```
Comment (jeff-phillips-18) [inline packages/module/src/pipelines/components/nodes/TaskNode.tsx:133]: By default we only want highlight the failed and cancelled statuses when details are hidden. Apps that want to show others can add them but we should keep the default setting as it was.

Comment (karthikjeeyar) [reply packages/module/src/pipelines/components/nodes/TaskNode.tsx]: Reverted this change.

**Comment by jeff-phillips-18** [comment]
@andrew-ronaldson @serenamarie125 Please take a look. Surge link is above.

**Comment by jeff-phillips-18** [comment]
@andrew-ronaldson @mceledonia Can you please take a look at confirm these match the latest spec? I know @mceledonia was go to take a look based on contrast checker issues.

**Comment by andrew-ronaldson** [comment]
Here's some notes based on a conversation with @mceledonia 

1. We can consolidate the Running and inProgress tasks from the demo and use the alt-sync icon. For running we went with blue-200 and black-900 text for the default state and to bump up the contrast onSelect we matched both the icon & text to be black-900 (screenshot). 
2.  The Warning states was already using the black text onSelect so we applied the same treatment for consistency and better contrast on the gold.
Screen Shot 2023-04-17 at 5 41 11 PM

**Comment by karthikjeeyar** [comment]
Thanks @andrew-ronaldson. I have unified inprogress and running status icons and styles and updated warning selected state to be black. Please take a look

https://user-images.githubusercontent.com/9964343/232853197-959ff861-ee70-4f60-a26a-dc5ae088f403.mov

**Comment by andrew-ronaldson** [comment]
Thanks for this @karthikjeeyar this looks great!
I forgot to mention that I don't think the pending task needs to blue icon. I think that can remain the default grey.

**Comment by karthikjeeyar** [comment]
Pending Icon - Color used  `--pf-global--secondary-color--100`. @andrew-ronaldson Let me know if you have some other color defined for pending icon.

Light Mode:
 | default state                                |                           selected state |
-------------------------------| ------------------------ |
Screenshot 2023-04-18 at 11 39 32 PM|Screenshot 2023-04-18 at 11 39 47 PM

Dark mode: 
 | default state                                |                           selected state |
-------------------------------| ------------------------ |
Screenshot 2023-04-18 at 11 40 01 PM|Screenshot 2023-04-18 at 11 40 07 PM

**Review by andrew-ronaldson — APPROVED** [review]
LGTM, thanks

**Review by jeff-phillips-18 — APPROVED** [review]

**Review by jenny-s51 — COMMENTED** [review]
This is looking good! Just a quick question... in dark mode, should these two nodes both have a white icon and white text when in the selected state? Looks like there are some discrepancies

Screenshot 2023-04-19 at 9 19 29 AM
Screenshot 2023-04-19 at 9 19 33 AM

**Comment by andrew-ronaldson** [comment]
Just thinking about this some more following @jenny-s51 screenshots. Do the pending, idle and skipped tasks need a special styling beyond the icons? I can't remember if they had unique features prior to the PF update

**Review by andrew-ronaldson — COMMENTED** [review]
I noticed in the demo that the default Selected task has a lighter blue and white text. This can be the same primary blue in both light and dark theme.

The Skipped and Idle tasks had a problem with color contrast. I am using the same updated styling which is more consistent with the other statuses. 

**Light Default** - border: pf-color-black--600 | fill: pf-color-white | text : pf-color-black-900 | Icon: Default
**Light Hover** - border: pf-color-black--600 | border-size: 2px |  fill: pf-color-white | text : pf-color-black-900 | Icon: Default
**Light Selected** - border: pf-color-black--600 | fill: pf-color-black--600 | text & icon: pf-color-white

**Dark Default** - border: pf-color-black--300 | fill: pf-color-black--600 | text & icon : pf-color-white
**Dark Hover** - border: pf-color-black--300 | border-size: 2px |  fill: pf-color-black--600 | text & icon: pf-color-white
**Dark Selected** - border: pf-color-black--300 | fill: pf-color-black--300 | text & icon: pf-color-black--900

Screen Shot 2023-04-20 at 10 58 12 AM

**Comment by karthikjeeyar** [comment]
Updated

Screenshot 2023-04-21 at 7 39 22 PM Screenshot 2023-04-21 at 7 39 26 PM
Screenshot 2023-04-21 at 7 39 34 PM Screenshot 2023-04-21 at 7 39 40 PM

**Review by andrew-ronaldson — APPROVED** [review]
Thanks for these changes. LGTM

**Review by jeff-phillips-18 — APPROVED** [review]

**Review by jenny-s51 — APPROVED** [review]
👍

**Pull request merged**